### PR TITLE
A few WX phoenix related changes

### PR DESCRIPTION
**Pull request by jenshnielsen**
* Two additional methods have changed name in WX Phoenix. We should probably do consider doing this in a more elegant way but for now these are the minimal changes to make zoom work probably.

* Don't depend on wxversion in setupext. It's no longer a dependency of the wx backend so neither should it be here.

### CODE REVIEW

**Comment by WeatherGod** [comment]
attn: @wernerfb

**Comment by OceanWolf** [comment]
Just adding myself and @fariza to the list of participants as we need to follow this for MEP22.

File: lib/matplotlib/backends/backend_wx.py [hunk 1]
```diff
@@ -1713,14 +1713,20 @@ def draw_rubberband(self, event, x0, y0, x1, y1):
         rubberBandColor = '#C0C0FF'  # or load from config?
 
         # Set a pen for the border
-        color = wx.NamedColour(rubberBandColor)
+        if wxc.is_phoenix:
+            color = wx.Colour(rubberBandColor)
```
Comment (wernerfb) [inline lib/matplotlib/backends/backend_wx.py]: NamedColour is defined in wx_compat, so you could just use wxc.NamedColour(rubberBandColor)

File: lib/matplotlib/backends/backend_wx.py [hunk 2]
```diff
@@ -1713,14 +1713,20 @@ def draw_rubberband(self, event, x0, y0, x1, y1):
...
+            color = wx.NamedColour(rubberBandColor)
         dc.SetPen(wx.Pen(color, 1))
 
         # use the same color, plus alpha for the brush
         r, g, b, a = color.Get(True)
         color.Set(r, g, b, 0x60)
         dc.SetBrush(wx.Brush(color))
-        dc.DrawRectangleRect(rect)
+        if wxc.is_phoenix:
+            dc.DrawRectangle(rect)
```
Comment (wernerfb) [inline lib/matplotlib/backends/backend_wx.py]: Maybe define that in wx_compat?

Comment (jenshnielsen) [inline lib/matplotlib/backends/backend_wx.py]: Done, this require Monkey patching wx.ClientDC in the wx_compat

**Pull request merged**